packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -64,8 +80,14 @@ export default class RatingBar extends PureComponent {
         }
         {...props}
       >
+        {ratingValue && this.ratingLegend}
         {this.ratingItems}
-        {label &&
+        {ratingValue && 
+          <div className={theme.Label}>
+            {score}
+          </div>
+        }
+        {label && 
```

Comment:
Can I just pass in "18 of 30" as my label, instead of coding this special case to meet one particular use? Lets try to keep this component generic so developers can use it in a variety of circumstances.